src/PyCall.jl

```diff
@@ -298,6 +312,14 @@ function setindex!(o::PyObject, v, s::Union{Symbol,AbstractString})
     o
 end
 
+function setproperty!(o::PyObject, s::Symbol, v)
+    if s == :o
+        setfield(o, s, v)
```

Comment:
This should be depecated in favor of explicit `setfield` calls.   Most PyCall users should never modify the internal `.o` field.